homeassistant/components/smarty/fan.py

```diff
@@ -76,6 +76,16 @@ def speed(self) -> str:
         """Return speed of the fan."""
         return self._speed
 
+    def set_speed(self, speed: str) -> None:
```

Comment:
```suggestion
    def set_speed(self, speed: str):
```

Reply:
We prefer complete typing or not typing a function at all. So either include the return type or remove `str` typing of `speed`.

Reply:
Would it be useful to add this to the style guidelines?

Reply:
Yes. :+1:

Reply:
I will make a PR 😄

Reply:
Hi, it's not clear for me. Do I change it or leave it like this? Thanks

Reply:
You can leave it like this :)

Reply:
great, thanks